pkg/linter/linterExecute.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package linter
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/vpcmodel"
+)
+
+const issues = " issues:"
+
+// LinterExecute executes linters one by one
+// todo: mechanism for disabling/enabling lint checks
+func LinterExecute(configsMap map[string]*vpcmodel.VPCConfig) (issueFound bool, resString string) {
+	strPerVpc := []string{}
+	for _, config := range configsMap {
```

Comment:
There is a design decision here: linters can only look at one config at a time.
Might there be a linter that needs to consider multiple configs together?

Reply:
1. I don't have one in mind
2. I would expect lint to be relatively local
3. And yet, better have a flexible framework. I will change the interface as you suggest.

Reply:
done